app/controllers/application_controller.rb

```diff
@@ -50,6 +50,8 @@ def check_privacy_policy_accepted
     policy = PrivacyPolicy.current
     return if policy.nil?
 
+    return if request.format == "application/json"
```

Comment:
```suggestion
    return if request.format.json?
```

Does this work? We use a few mime types for JSON, and this should capture them all